src/audio/igo_nr/igo_nr.c

```diff
@@ -546,48 +609,57 @@ static void igo_nr_process(struct comp_dev *dev,
 	comp_update_buffer_produce(sink, sink_bytes);
 }
 
-static void igo_nr_get_igo_params(struct comp_dev *dev)
+static void igo_nr_print_config(struct comp_dev *dev)
```

Comment:
why are you passing in dev? you have cd already in caller function

Reply:
`dev` is called as the first argument of `comp_dbg`.

Reply:
ack